app/views/categories/index.html.erb

```diff
@@ -0,0 +1,34 @@
+
+ <div class="flex justify-between items-center h-11 bg-[#3778c2] text-white"> 
+    <div class="ml-4 cursor-pointer">
+      <%= link_to "<", authenticated_root_path %>
+    </div>
+    <h2 class="uppercase">
+      Categories
+    </h2>
+    <div class="actions mr-4 cursor-pointer text-2xl">
+      <%= link_to "+", new_category_path %>
+    </div>
```

Comment:
- [ ] Please implement the "Add" button to be at the bottom of the design to better match the original design.

![image](https://github.com/codedit334/budget-app/assets/62290545/10e986a6-8db6-4f34-b421-222c2600c56f)